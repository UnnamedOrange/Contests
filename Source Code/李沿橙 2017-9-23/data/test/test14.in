5
37
589 564 999 993 48 539 574 702 458 611 855 981 981 245 393 846 405 734 322 590 52 748 899 833 314 500 351 491 980 584 244 750 472 324 446 456 398 
47
424 238 893 949 189 639 451 373 174 570 48 963 586 378 205 564 802 747 108 817 152 909 353 119 178 906 670 583 650 629 109 436 735 116 821 354 147 880 633 198 750 489 432 565 547 668 412 
49
688 512 187 997 199 619 294 360 918 245 830 296 682 155 570 142 653 430 825 531 280 114 125 400 415 911 997 532 192 745 468 609 109 875 186 895 112 596 42 566 820 387 200 458 269 248 988 289 738 
30
853 747 880 639 109 596 396 405 801 657 720 768 463 72 674 807 700 330 940 242 917 449 513 912 774 224 989 143 10 301 
35
902 177 898 794 741 384 47 160 375 15 833 995 637 729 521 128 103 567 575 229 146 867 7 742 209 774 199 175 464 972 308 976 457 250 301 
