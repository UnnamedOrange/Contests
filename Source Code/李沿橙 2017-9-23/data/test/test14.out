48 52 245 314 322 324 351 393 398 446 456 458 472 491 500 539 564 574 584 589 611 702 734 748 833 846 855 980 981 981 993 999 899 750 590 405 244 
48 108 109 116 119 147 152 174 178 198 205 353 354 373 412 424 432 436 451 547 564 565 570 583 586 629 639 650 668 670 735 747 802 817 821 880 906 909 949 963 893 750 633 489 378 238 189 
42 109 112 114 142 155 186 187 192 199 245 248 269 280 294 296 387 400 415 458 468 531 532 566 570 609 619 653 688 738 820 825 875 895 911 988 997 997 918 830 745 682 596 512 430 360 289 200 125 
10 72 109 224 301 396 405 449 513 639 657 674 700 747 768 774 801 807 880 912 917 989 940 853 720 596 463 330 242 143 
7 15 47 103 128 146 175 177 199 209 229 250 301 375 384 464 521 567 637 741 742 774 794 833 898 902 972 976 995 867 729 575 457 308 160 
